### ISSUE TITLE
Makefile - copy binary bug

### ISSUE THREAD

**Issue by criadoperez** [maticnetwork/polygon-cli#85]
The [Makefile file](https://github.com/maticnetwork/polygon-cli/blob/f1b943f22f560fb617570be9911505a0a1b72840/Makefile#L40) has the following code:

```
.PHONY: install
install: build ## Install the go binary.
        $(RM) $(INSTALL_DIR)/$(BIN_NAME)
        cp $(BUILD_DIR)/$(BIN_NAME) $(INSTALL_DIR)/
```

Which produces at the end of the `make install` command, run something like this:

```bash
rm -f ~/go/bin/polycli
cp ./out/polycli ~/go/bin
```
If the `bin` folder doesn't exist, which would be the case in a clean system, this code renames the executable file `polycli` to `bin` instead of copying the file into the `bin` folder which would be the intended intention of the code.

In other words you end up with the executable `~/go/bin` instead of `~/go/bin/polycli`.

**Comment by praetoriansentry** [comment]
Thanks @criadoperez  for the fix in #86
